src/psyclone/generator.py

```diff
@@ -437,6 +437,12 @@ def main(arguments):
     parser.add_argument(
         '-I', '--include', default=[], action="append",
         help='path to Fortran INCLUDE or module files')
+    parser.add_argument(
+        '--enable-cache', action="store_true", default=False,
+        help='whether to enable caching imported module dependencies (if '
```

Comment:
Please update as per my comments in the corresponding documentation file.

Reply:
Done